js_modules/dagster-ui/packages/ui-core/src/assets/insights/AssetCatalogRateCard.tsx

```diff
@@ -9,19 +9,66 @@ export interface AssetCatalogRateCardProps {
   value: number | null; // e.g., 0.981 for 98.1%
   prevValue: number | null; // e.g., 1 for 100%
   loading: boolean;
+  unit: 'percent' | 'seconds';
+}
+
+function pct_change(x: number, y: number): number {
+  if (x === 0) {
+    if (y === 0) {
+      return 0.0; // No change
+    }
+    return Number.POSITIVE_INFINITY;
+  }
+
+  if (y > x) {
+    return Math.round(1000 * (y / x - 1)) / 10;
+  }
+  return Math.round(-(1000 * (1 - y / x))) / 10;
```

Comment:
If you're aiming to get a specific number of places after the decimal, I think you'll want to handle this via string formatting, e.g. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat.

You can use the `roundingMode` option to get the rounding behavior you want: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat#roundingmode.